Created classes for EventCatalogAPI.

sstambat12 opened the pull request:
Created basic class structure for creating tables in EventCatalogAPI as per my understanding.

LeelaCreations committed:
Added classes to create tables  for EventCatalogAPI

sstambat12 committed:
Created classes for EventCatalogAPI.

LeelaCreations commented:
looks good to me, but I feel, let us not migrate the code to local SQL DB as we eventually use the Docker not the local one.

still I feel like having separate table for organizer and Location table should contain more details like address of the events.

LeelaCreations committed:
removed the Organizer class and added Organizer name and description props under EventsCatalog class

LeelaCreations committed:
Added wwwroot folder and 20 event pics

LeelaCreations committed:
Added Seed class

LeelaCreations committed:
Added Location table and also added FK in Events Table

Commit:
Merge remote-tracking branch 'origin/leela' into sujata

Commit:
Fixing compilation errors after merging from leela

After I merged changes from leela branch, I got some compilation errors. Most of them were related to namespace change to EventCatalogAPI from EventCatalog.

Commit:
Created Pics Folder

Created New folder Pics under wwwroot and moved images in it.

Commit:
Added PicController API to get Images from wwwroot

Commit:
Seeding Database

Called CatalogSeed's seed method to seed the database.

Commit:
Removing Unnecessary Migration files.

Commit:
Added CatalogController API

Added few APIs in it to get all the events, get all the event types, event categories and event locations. And one API to filter events according to type, category and location

Commit:
Added Swagger to create documentation.
